semgrep-core/tests/OTHER/rules/regexp_capture_groups.yaml

```diff
@@ -0,0 +1,7 @@
+rules:
+  - id: new-Gemfile-dependency
+    languages:
+      - generic
+    message: New dependency with name $1 and version $2
+    pattern-regex: gem "(.*)", "(.*)"
```

Comment:
Would be cool if one could use `metavariable-pattern` on `$1` etc, but I'm not sure the `$1` will parse as a valid metavariable name? It needs to start with a letter.

Reply:
hmm right. Not sure we check that though when we parse metavariable-pattern.